Static/Row.swift

```diff
@@ -134,7 +140,12 @@ public struct Row: Hashable, Equatable {
     // MARK: - Initializers
 
     public init(text: String? = nil, detailText: String? = nil, selection: Selection? = nil,
-        image: UIImage? = nil, accessory: Accessory = .None, cellClass: CellType.Type? = nil, context: Context? = nil, editActions: [EditAction] = [], UUID: String = NSUUID().UUIDString) {
+        image: UIImage? = nil, accessory: Accessory = .None, cellClass: CellType.Type? = nil, cellNib: UINib? = nil, height: CGFloat? = nil, context: Context? = nil, editActions: [EditAction] = [], UUID: String = NSUUID().UUIDString) {
+        
+        if let _ = cellNib where cellClass != nil {
```

Comment:
Gotcha the debugging part makes sense. There’s a but with the `nil` check, see https://github.com/venmo/Static/pull/45#discussion_r42311119 please. :pray: